Project/demo/src/main/java/com/example/demo/model/User.java

```diff
@@ -0,0 +1,75 @@
+    package com.example.demo.model;
+
+    import javax.persistence.*;
+
+    @Entity
+    @Table(name = "userregister")
```

Comment:
We need this table for saving users, so this table should be named "user" instead.